dotnet/src/Connectors/Connectors.AI.Oobabooga/TextCompletion/TextCompletionRequest.cs

```diff
@@ -173,5 +174,5 @@ public sealed class TextCompletionRequest
     /// In addition to the defaults. Written between "" and separated by commas. For instance: "\nYour Assistant:", "\nThe assistant:"
     /// </summary>
     [JsonPropertyName("stopping_strings")]
-    public List<string> StoppingStrings { get; set; } = new List<string>();
+    public IList<string> StoppingStrings { get; set; } = new List<string>();
```

Comment:
Can we make it so this will also read from `stop_sequences` or is that the responsibility of the plugin developer | planner developer to do:

```
ExtensionData = new Dictionary<string, object>()
{
  { "Temperature", 0.0 },
  { "StopSequences", new[] { StopSequence } }, // to support openai
  { "StoppingStrings", new[] { StopSequence } }, // to support oogabooga
  { "MaxTokens", this.Config.MaxTokens ?? 1024 },
}
```

Reply:
One problem with making this the responsibility of the connector is that every connector needs to be aware of every other connector so that it can perform this conversion. And if a new AI and connector appears existing connectors will need an update.

The pattern you have proposed above is the short term solution and will work with the changes in this PR.

The medium term goal is to support the ability to configure requesting settings on a per model/provider basis. So I should be able to configure the Oobabooga and OpenAI settings separately when I create a Semantic Function.

Reply:
I would not dictate this to be a `IList` we can use `ICollection`